src/spryker/harness.yml

```diff
@@ -123,7 +123,7 @@ attributes:
       DE: = 'zed-de-' ~ @('hostname')
       AT: = 'zed-at-' ~ @('hostname')
       US: = 'zed-us-' ~ @('hostname')
-    root_dir: = @('app.web_directory') ~ '/Zed'
+    root_dir: "= @('spryker.demoshop-version') >= '202108.0'? @('app.web_directory') ~ '/Backoffice' : @('app.web_directory') ~ '/Zed'"
```

Comment:
Actually, it's better to use the `version_compare` function here. Example: https://github.com/inviqa/harness-base-php/blob/1.5.x/src/spryker/harness.yml#L92

Reply:
k, changed it